materials/vgui/ttt/loadingscreen/loading.html

```diff
@@ -0,0 +1,11 @@
+<html>
+	<head>
+		<title>Loading..</title>
+		<link rel="stylesheet" type="text/css" href="loading.css?3" />
+		<script type="text/javascript" src="js/thirdparty/jquery.js"></script>
```

Comment:
Why do you need jQuery here?

Reply:
okay, after talking about this, I think this can stay in here since vanilla TTT also has this included